tests/integration/test_integration_all_rust.py

```diff
@@ -909,7 +909,8 @@ def test_hello_echo(self):
         yield client.connect()
         result = yield client.hello()
         assert result != {}
-        assert result["use_webpush"] is True
```

Comment:
These are all asserting the field exists in the responses so I think we want to put all these back (basically revert the whole file) as well